libraries/Microsoft.Bot.Builder.Ai/LocaleConverter.cs

```diff
@@ -171,11 +127,11 @@ private List<TextAndDateTime> extractDate(string message, string fromLocale)
         }
 
         //Convert a message from locale to another locale
-        public Task<string> Convert(string message, string fromLocale, string toLocale)
+        public async Task<string> Convert(string message, string fromLocale, string toLocale)
         {
             List<TextAndDateTime> dates = extractDate(message, fromLocale);
             if (dates.Count == 0)
-                return Task.FromResult(message);
+                return  message;
```

Comment:
This doesn't make sense to me. I'm a developer calling:
.Convert("Hello", "en-us", "jp-jp"). 

In this code, if there are no dates, I would get back "Hello". 

I'm confused as to what this is supposed to be doing.